app.py

```diff
@@ -45,9 +45,9 @@ def get_account_roles(user_id):
     return {'account_roles': account_roles}
 
 
-def get_credentials(account_id: str, role_name: str):
-    uid = connexion.request.user
-    realm = connexion.request.token_info['realm']
```

Comment:
You could also use `flask.request.user` and `flask.request.token_info['realm']`. Anyway, using as parameters is better.